README.md

```diff
@@ -1,3 +1,5 @@
+### English | [简体中文](https://github.com/yunshuipiao/algorithms/blob/master/README_CN.md)
```

Comment:
I think making it smaller would be better. (normal font size)
example: https://github.com/kdn251/interviews

Reply:
@keon thank you .I will fix it .

Reply:
@keon fix it . I will keep up to date with main README.md and the Chinese part as far as I can.